src/test/java/org/osiam/client/LoginResourceOwnerPasswordCredentialsIT.java

```diff
@@ -61,6 +73,19 @@ public void login_with_resource_owner_password_credentials_grant_should_provide_
         assertNotNull("The refresh token was null.", accessToken.getRefreshToken());
     }
 
+    @Test
+    @DatabaseSetup("/database_seed_login_with_old_hash.xml")
+    public void login_updates_to_bcrypt_hash() throws SQLException {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/t-pick/connector4java-integration-tests/file/2113725304/issues/source?bid=3145228&fileBranchId=3145235#l78)